includes/Classifai/Providers/Azure/ComputerVision.php

```diff
@@ -610,6 +610,16 @@ protected function scan_image( string $image_url, \Classifai\Features\Feature $f
 					'Ocp-Apim-Subscription-Key' => $settings['api_key'],
 					'Content-Type'              => 'application/json',
 				],
+				/**
+				 * Filters the timeout for the image scan request.
+				 * Default: 60 seconds.
+				 *
+				 * @param int $timeout Timeout in seconds.
+				 */
```

Comment:
We'll want to ensure this doc block follows the JS Hook standards so it shows properly in our documentation. See the `classifai_should_smart_crop_image` filter in this file for an example